cmd/flowrunner/runner_test.go

```diff
@@ -78,18 +79,25 @@ type runResult struct {
 }
 
 func runFlow(env utils.Environment, assetsFilename string, contactFilename string, flowUUID flows.FlowUUID, callerEvents [][]flows.Event) (runResult, error) {
-	assetsJSON, err := readFile("flows/", assetsFilename)
-
+	// load both the test specific assets and default assets
+	defaultAssetsJSON, err := readFile("", "default.json")
```

Comment:
Moving some assets into a default file so they don't have to be duplicated in every test